.github/workflows/r_check.yaml

```
@@ -50,6 +50,12 @@ jobs:
           # args: 'c("--ignore-vignettes")'
           # build_args: 'c("--no-build-vignettes")'
 
+      - name: slow tests
+        if: success() && contains('refs/heads/main', github.ref)
+        run: |
+          Rscript -e 'remotes::install_github("kkholst/targeted")'
+          make slowtest
```

I'm quite sure that the workflow won't fail for failing tests because the R command inside the make target does returns with status code 0, independent whether tests inside `inst/slowtest` fail. 

In trialsim we use
```
slowtest:
	@echo 'res <- tinytest::run_test_dir("inst/slowtest"); if (summary(res)["Total", "fails"] > 0) {print(res); quit(status = 2)}' | $(R)
```
to let the workflow fail if any of the tests fail. I'm not changing it now because we can first think about a more elegant way to do this.

---

I will fix this.